analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -72,8 +70,12 @@ def get_analyzer_checkers(cls, cfg_handler, environ):
 
         try:
             command = shlex.split(' '.join(command))
```

Comment:
Here again, is it a good idea to join and split instead of keeping the command as the original list?

Reply:
See my comment before.